### DISCUSSION TITLE
JBC tutorial error

### DISCUSSION THREAD

**Discussion by zqliu012345** [mdolab/dafoam#151] [Tutorials]
![image](https://user-images.githubusercontent.com/64395492/122540075-dab71e00-d05a-11eb-8fcb-f008bdadb21d.png)
when I run the JBC  tutorial, at the coloring Solver step, it shows the "segmentation fault ( core dumped)"

**Comment by friedenhe** [comment]
It is likely that you don't have enough memory to run this case. DAFoam is memory intensive, check [this](https://dafoam.github.io/mydoc_get_started_faq.html#how-many-cpu-cores-to-use-and-how-much-memory-does-it-need)
